src/client/api/v1/get_balance.h

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __CLIENT_API_V1_BALANCE_H__
+#define __CLIENT_API_V1_BALANCE_H__
+
+#include <stdint.h>
+
+#include "client/api/v1/response_error.h"
+#include "client/client_service.h"
+#include "core/address.h"
+#include "core/types.h"
+
+typedef struct {
+  char address[IOTA_ADDRESS_HEX_BYTES + 1];  // with null terminator
+  uint16_t max_results;
+  uint16_t count;
+  uint64_t balance;
+} get_balance_t;
```

Comment:
memory alignment needs to be considered.
```suggestion
typedef struct {
  uint16_t max_results;
  uint16_t count;
  uint64_t balance;
  char address[IOTA_ADDRESS_HEX_BYTES + 1];  // with null terminator
} get_balance_t;
```

Reply:
I understand memory alignment in terms of C language.
But what is the reason for `address` to be the last struct member?

Reply:
could you elaborate how can memory aligned if you put `address` at beginning?

Reply:
I understand we save memory, so the change is positive.
However would it cause bugs if `address` was in the beginning?

Reply:
yes, It could cause bugs at runtime. anyway it's not a problem if you know what you do to this struct, like I mentioned above  direct access it via memory address is risky.